fix: broken list format and typo fixes on workshop (#103)
thoth-tech/handbook: docs/learning/training/distinction-high-distinction-workshop.md

At this level, you will assist in building the pieces needed by your team to complete the puzzle. It
would help if you had support to complete your work to an acceptable standard or get the tasks done

> on time. Luckily, you have a team that will provide you with that support; make sure you don't let
> them down.

Reviewer: **Suggestion:** removes contraction. Sounds more positive and encouraging

```suggestion
on time. Luckily, you have a team that will provide you with that support; make sure you are doing
them proud.
```

### Credit

Show you are a responsible team member to receive a credit.

At this level you will be building the pieces needed by your team to complete the puzzle. You work
at this level if you are a responsible team member. You can follow instructions from your lead and

> deliver the designated work at an acceptable standard in the allocated timeframe. You must show:

Reviewer: reword:
```suggestion
deliver the designated work at an acceptable standard in the allocated timeframe. You must:
```

> - Clear contributions
> - Be engaged and present
> - Responsible

Reviewer: Reword: related to above
```suggestion
- Show clear contributions
- Be engaged and present
- Be responsible
```
